modular_zubbers/modules/tesh-vali_augments/code/modules/research/techweb/all_nodes.dm

```diff
@@ -0,0 +1,14 @@
+//teshvali robolimb research node
+
+/datum/techweb_node/teshvalicyber
+	id = "teshvalicyber"
+	display_name = "Raptoral Cybernetics"
+	description = "Specialized cybernetic limb designs, courtesy of Ceremi Cybernetics."
```

Comment:
```suggestion
	description = "Specialized cybernetic limb designs, courtesy of Ceremi Cybernetics."
	prereq_ids = list("base")
```

Reply:
This will make it so your new research actually shows up on the research menu

Reply:
I'm not too concerned about that tbh